src/components/ItemsSection.tsx

```diff
@@ -41,12 +41,14 @@ const ItemsSection = ({ title, itemsData, category }: ItemsSectionProps) => {
       const newItem = {
         ...item,
         id: itemId,
-        name: item.name,
         image: item.image || '/placeholder.jpg',
         stats: item.stats || { strength: 0, power: 0 },
```

Comment:
Te dwie "alternatywne" wartości możesz gdzieś zapisać powyżej komponentu LUB gdzieś w globalnym folderze constants, pod warunkiem że korzystasz z nich w wielu plikach. Jeśli nie, to ponad komponentem
`const DEFUALT_IMAGE`
`const DEFAULT_STATS`